holmes/main.py

```diff
@@ -194,8 +194,9 @@ def alertmanager(
         None,
         help="Investigate all alerts with this name (can be regex that matches multiple alerts). If not given, defaults to all firing alerts",
     ),
-    alertmanager_label: Optional[str] = typer.Option(
-        None, help="For filtering alerts with a specific label. Must be of format key=value"
+    alertmanager_label: Optional[List[str]] = typer.Option(
+        [],
+        help="For filtering alerts with a specific label. Must be of format key=value. If --alertmanager-label is passed multiple times, alerts must match ALL labels"
```

Comment:
not sure what this means
`If --alertmanager-label is passed multiple times`

Reply:
You can run `holmes investigate alertmanager --alertmanager-label x=y --alertmanager-label a=b`
Any ideas on how to phrase better?